.github/issue_template.md

```diff
@@ -50,6 +50,11 @@
 ```
 (paste your output here)
 ```
+#### Output of `show tech support`:
```

Comment:
Should read `show techsupport` (no space)

Reply:
addressed